### PR TITLE
React to dotnet/cli/issues#1970

### PR DESCRIPTION
**Pull request by prafullbhosale**
cc @abpiskunov, @barrytang 

This change needs to wait till the CI machines start using a newer version of dotnet CLI for building.

### CODE REVIEW

**Comment by prafullbhosale** [comment]
cc @Eilon @DamianEdwards (for Rc2 ask mode approval)

**Comment by prafullbhosale** [comment]
This change can be merged now that the build scripts are using cli version 2202 
cc @davidfowl

**Comment by davidfowl** [comment]
That's not right though. It's using the wrong target framework.

File: src/dotnet-aspnet-codegenerator/project.json [hunk 1]
```diff
@@ -7,15 +7,20 @@
     "Microsoft.Extensions.CodeGeneration": "1.0.0-*",
     "Microsoft.Extensions.CommandLineUtils": "1.0.0-*",
     "Microsoft.Extensions.DependencyInjection": "1.0.0-*",
-    "NETStandard.Library": "1.5.0-*",
     "System.Diagnostics.Process": "4.1.0-*"
   },
   "frameworks": {
-    "netstandardapp1.5": {
```
Comment (davidfowl) [inline src/dotnet-aspnet-codegenerator/project.json]: Why this change?

File: src/dotnet-aspnet-codegenerator/project.json [hunk 2]
```diff
@@ -7,15 +7,20 @@
...
+    "netstandard1.5": {
       "imports": [
         "dnxcore50",
         "portable-net45+win8"
-      ]
+      ],
+      "dependencies": {
```
Comment (davidfowl) [inline src/dotnet-aspnet-codegenerator/project.json]: Why did you move this here? Just put it with the rest of the dependencies.

File: src/dotnet-aspnet-codegenerator/project.json (hunk 1 above)
Comment (prafullbhosale) [inline src/dotnet-aspnet-codegenerator/project.json]: I thought to make it a portable app, the tfm was supposed to be netstandard1.5 as per 
https://github.com/dotnet/cli/blob/rel/1.0.0/TestAssets/TestProjects/PortableTests/PortableApp/project.json

Isn't this eventually going to be netcoreapp? 
Not sure if it already needs to be netcoreapp?

**Comment by prafullbhosale** [comment]
Tagging aspnet/Release/issues/4

**Comment by Eilon** [comment]
@davidfowl are you signed off on this? If so, approved for RC2.

**Comment by prafullbhosale** [comment]
Ping. @davidfowl

**Comment by prafullbhosale** [comment]
Spoke to @davidfowl offline. 
He said its ok. 
So merging it to release first and then to dev.

**Comment by prafullbhosale** [comment]
066beb7db81d5299f0dc455d0bf9a6410cf89de2
